src/mesh/mesh_communication_global_indices.C

```diff
@@ -858,9 +889,24 @@ void MeshCommunication::find_global_indices (const Parallel::Communicator & comm
             global_ids.push_back (cast_int<dof_id_type>(std::distance(my_bin.begin(), pos) + my_offset));
           }
 
-        // and trade back
-        communicator.send_receive (procdown, global_ids,
-                                   procup,   filled_request[procup]);
+        // and send back
+        id_return_requests.push_back(Parallel::Request());
+	communicator.send (pid, global_ids,
```

Comment:
Is this line indented slightly differently than the line above (1 char)?

Reply:
Oh, hell.  I let a tab slip in somehow.  I'll fix it.  In a separate commit, if you don't mind, so I can merge with other branches easier.

Reply:
Oh, I get it.  I was editing code on a cluster where I haven't yet copied over my .vimrc.  There's a few tabs I need to fix.  Thanks for catching that one!